postgres/create_tables.sql

```diff
@@ -145,13 +145,13 @@ DECLARE
   wasPopping    filesystem.EntityID%type;
   oldEntity     filesystem.EntityID%type;
 BEGIN
-  SELECT filesystem.EntityID INTO rootID FROM filesystem WHERE Parent = 0;
+  SELECT filesystem.EntityID INTO rootID FROM filesystem WHERE EntityID = uuid_nil();
   
-  newEntity := (SELECT new_entity(2, 'downloads'::VARCHAR, 1, false));
```

Comment:
Double check the logic here, I wasn't too sure what the magic number 2 meant, and what the purpose of this was